rockr/views.py

```diff
@@ -93,7 +94,63 @@ def user_goals(user_id):
         ).first()
         db_manager.delete(ug)
         return format_response(204, None)
-
+    
+#Added for the User Band queries
+@app.route('/user_band/<int:user_id>', methods=["GET","POST","DELETE"])
+def user_band(user_id):
+    if request.method == "GET":
+        umi = UserBand.query.filter_by(user_id=user_id)
+        bandsids = [Band.query.get(b.band_id) for b in umi]
+        return format_response(200, serialize_query_result(bandsids))
+    #For a new User Band Adding Query
+    elif request.method == "POST":
+        ub = UserBand.query(func.max(UserBand.id))
+        highestid = ub.id;
+        highestid = highestid+1;
+        ubnew = {
+            "id": highestid,
+            "user_id": user_id,
+            "band_id": int(request.args["id"])
+        }
+        db_manager.insert(ubnew)
```

Comment:
you could just do `db_manager.insert(UserBand(user_id=user_id, band_id=request.args["id"])`. You don't need to cast the id to an int either